core/src/main/resources/org/apache/spark/ui/static/executorspage.js

```diff
@@ -160,6 +170,149 @@ function reselectCheckboxesBasedOnTaskTableState() {
   }
 }
 
+var executorSummaryMetricsTableArray = [];
+var executorSummaryMetricsTableCurrentStateArray = [];
+var executorSummaryMetricsDataTable;
+
+function getColumnNameForExecutorMetricSummary(columnKey) {
+  switch(columnKey) {
+    case "JVMHeapMemory":
+      return "JVM Heap Memory";
+
+    case "JVMOffHeapMemory":
+      return "JVM Off Heap Memory";
+
+    case "OnHeapExecutionMemory":
+      return "On Heap Execution Memory";
+
+    case "OffHeapExecutionMemory":
+      return "Off Heap Execution Memory";
+
+    case "OnHeapStorageMemory":
+      return "On Heap Storage Memory";
+
+    case "OffHeapStorageMemory":
+      return "Off Heap Storage Memory";
+
+    case "OnHeapUnifiedMemory":
+      return "On Heap Unified Memory";
+
+    case "OffHeapUnifiedMemory":
+      return "Off Heap Unified Memory";
+
+    case "DirectPoolMemory":
+      return "Direct Pool Memory";
+
+    case "MappedPoolMemory":
+      return "Mapped Pool Memory";
+
+    case "ProcessTreeJVMVMemory":
+      return "Process Tree JVM Memory";
```

Comment:
This should be VMemory

Reply:
But there are `ProcessTreeJVMVMemory ` and `ProcessTreePythonVMemory `

Reply:
I am talking about the column heading, it should be `Process Tree JVM VMemory`.

Reply:
> I am talking about the column heading, it should be `Process Tree JVM VMemory`.

Hmmm, sorry for my mistake...

Reply:
> I am talking about the column heading, it should be `Process Tree JVM VMemory`.

updated